help/en/html/hardware/openlcb/index.shtml

```diff
@@ -69,12 +69,39 @@
       INACTIVE shortly after. This can be used for events that indicate momentary things on the
       layout like "it's noon".</p>
 
+      <p>When "Show System-specific columns" is checked in the table, there are two additional
+      columns shown that are specific to OpenLCB Sensors:
+      <dl>
+        <dt>Authoritative</dt><dd>Normally checked, this means that the JMRI Sensor
+                                  will report its state when a query comes from another
+                                  OpenLCB node.  That will then allow that other ndoe
+                                  to adjust its own state to be consistent.
+                                  <p>
+                                  When not checked, JMRI will report the state of the
+                                  Sensor as "unknown".  This will allow some other
+                                  user of these events to be the authoritative
+                                  source.
+                                  </dd>
+        <dt>Always listen</dt><dd>Normally checked, this means that the JMRI Sensor
```

Comment:
I think this is not the correct understanding of the behavior.

We have a tooltip in the bundle, which says the following:
```
OlcbStateListenHeader = Always listen
OlcbStateListenToolTip = If ticked, all state reports will update local state. If unticked, state \
  reports will update local state only if local state is UNKNOWN. State change messages (Event \
  Reports) always change local state. Might be helpful to untick if the state flip-flops multiple \
  times after a query, which means that the event does not have a consistent distributed state.
```

I have not managed to find how this tooltip is displayed to the user. (It is possible that it is not displayed at all, which would be a bug worth to fix.)

So ultimately this tickbox only matters to interpreting "producer identified known/unknown" and "consumer identified known/unknown" messages.

Typically when Is Authoritative is true,  then this is better unchecked.
If there is only one authoritative source for an event state, then having this checked is good, because it will make JMRI converge to the correct state irrespectively of how long JMRI has been running. This is particularly important if the authoritative producer / consumer of the event does not save its state between reboots. 
Then there is a case where you have two momentary pushbuttons for setting a turnout, and those input lines believe that they are authoritative and confuse JMRI (because typically both are invalid) -> then this needs to be unchecked.

Reply:
Thank you for the detailed comments!  I've updated this from your comments.